src/commands/deployAzurePipeline/configureCicdPipeline/configurePipeline.ts

```diff
@@ -0,0 +1,47 @@
+import * as vscode from 'vscode';
+import * as k8s from 'vscode-kubernetes-tools-api';
+import { IActionContext } from 'vscode-azureextensionui';
+import { resourceNode } from './models/resourceNode';
+const CONFIGURE_PIPELINE_COMMAND = 'configure-cicd-pipeline';
+const DEPLOY_TO_AZURE_EXTENSION_ID = 'ms-vscode-deploy-azure.azure-deploy';
+
+export async function configurePipeline(context: IActionContext, target: any): Promise<void> {
+    const deployToAzureExtensionInstalled = await isDeployToAzureExtensionInstalled();
+    if (deployToAzureExtensionInstalled) {
+        const cloudExplorer = await k8s.extension.cloudExplorer.v1;
+        if (cloudExplorer.available) {
+            const clusterTarget = cloudExplorer.api.resolveCommandTarget(target);
+            if (clusterTarget && clusterTarget.cloudName === "Azure" && clusterTarget.nodeType === "resource" && clusterTarget.cloudResource.nodeType === "cluster") {
+                const cluster: resourceNode = {resource: clusterTarget.cloudResource, subscriptionId: clusterTarget.cloudResource.subscription.subscriptionId};
```

Comment:
💡 No annotation is needed here.

Reply:
Isn't it good to have?

Reply:
:bulb: Why is it good? i.e. what are we achieving by the const level type annotation?  **Because** 

* By assigning `{ resource: blah, subscriptionId: blahs};` is achieving that purpose? **Or** please let me know if I missed a context. further you had similar discussion hereh: https://github.com/Azure/vscode-aks-tools/pull/42#discussion_r499922583